elb/status/status.go

```diff
@@ -0,0 +1,90 @@
+/*
+Package status provides an updater for an ELB frontend to update ingress statuses.
+*/
+package status
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/sky-uk/feed/controller"
+	"github.com/sky-uk/feed/elb"
+	"github.com/sky-uk/feed/k8s"
+	"github.com/sky-uk/feed/util"
+
+	aws_elb "github.com/aws/aws-sdk-go/service/elb"
+	log "github.com/sirupsen/logrus"
+)
+
+// Config for creating a new ELB status updater.
+type Config struct {
+	Region           string
+	LabelValue       string
+	KubernetesClient k8s.Client
+}
+
+// New creates a new ELB frontend status updater.
+func New(conf Config) (controller.Updater, error) {
+	session, err := session.NewSession(&aws.Config{Region: &conf.Region})
+	if err != nil {
+		return nil, fmt.Errorf("unable to create ELB status updater: %v", err)
+	}
+
+	return &status{
+		awsElb:           aws_elb.New(session),
+		labelValue:       conf.LabelValue,
+		kubernetesClient: conf.KubernetesClient,
+	}, nil
+}
+
+type status struct {
+	awsElb           elb.ELB
+	labelValue       string
+	kubernetesClient k8s.Client
+	loadbalancers    map[string]elb.LoadBalancerDetails
```

Comment:
Isn't load balancers a two word noun? Should be camel cased

Reply:
good spot :+1: